libs/model/src/policies/FarcasterWorker.policy.ts

```diff
@@ -99,7 +103,7 @@ export function FarcasterWorker(): Policy<typeof inputs> {
         // create onchain content from reply cast
         const contestManagers = [
           {
-            url: community.ChainNode!.private_url! || community.ChainNode!.url!,
+            url: getChainNodeUrl(community.ChainNode!),
```

Comment:
`community.ChainNode.url or `community.ChainNode.private_url` are already properly formatted using the getters on the Sequelize model. Using `getChainNodeUrl` here just abstracts away `||` so I don't think this will resolve your `NULL` bug. Do we have the associated community ID of the votes which caused errors?